Find-and-Replace/Replace-line-break-with-paragraph-mark/.NET-Standard/Replace-line-break-with-paragraph-mark/Program.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Syncfusion.DocIO;
+using Syncfusion.DocIO.DLS;
+using System.IO;
+
+namespace Replace_line_break_with_paragraph_mark
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            using (FileStream fileStreamPath = new FileStream(Path.GetFullPath(@"../../../Data/Input.docx"), FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
+            {
+                //Open an existing Word document.
+                using (WordDocument document = new WordDocument(fileStreamPath, FormatType.Docx))
+                {
+                    // Iterate through the paragraphs of Word document.
+                    foreach (WParagraph paragraph in document.Sections[0].Body.Paragraphs)
+                    {
+                        for (int i = 0; i < paragraph.Items.Count; i++)
+                        {
+                            Entity entity = paragraph.Items[i];
+                            //Set paragraph child element as break.                                                     
+                            if (entity.EntityType == EntityType.Break)
+                            {
+                                Break breakItem = entity as Break;
+                                //Replace line break with paragraph mark.
+                                if (breakItem.BreakType == BreakType.LineBreak)
```

Comment:
Separate this replace logic alone as separate method

ReplaceLineBreakWithPara()

Reply:
Addressed